docs/core/proto-docs.md

```diff
@@ -2750,7 +2750,7 @@ InterfaceImplementerDescriptor describes an interface implementer
 <a name="cosmos.base.reflection.v2alpha1.MsgDescriptor"></a>
 
 ### MsgDescriptor
-MsgDescriptor describes a Cosmos SDK message that can be delivered with a transaction
+MsgDescriptor describes a cosmos-sdk message that can be delivered with a transaction
```

Comment:
What's the rationale behind these changes? Is `Cosmos SDK` not OK to use?

Reply:
these are the changes reflected by `make proto-gen`